Tests/js/views/fields/assets/ez-selection-editview-tests.js

```diff
@@ -575,9 +575,101 @@ YUI.add('ez-selection-editview-tests', function (Y) {
         },
     });
 
+    regressionTest = new Y.Test.Case({
```

Comment:
Variable for test case should point somehow what is it related to - regression doesn't say much ;)